apps/aurora-portal/src/client/routes/gardener/-components/SearchBar.tsx

```diff
@@ -21,51 +20,41 @@ export const SearchBar = ({
   setSortTearm,
   sortTearm,
 }: SearchBarProps) => (
-  <div className="flex flex-col lg:flex-row justify-between items-stretch lg:items-center border-b border-aurora-gray-800 pb-4 mb-6 gap-4">
-    <div className="flex flex-grow items-center">
-      <div className="relative flex-grow">
-        <input
-          type="text"
-          placeholder="Search clusters..."
-          value={searchTerm}
-          onChange={(e) => setSearchTerm(e.target.value)}
-          className="w-full pl-9 pr-4 py-2 bg-aurora-gray-800 border border-aurora-gray-700 rounded-md text-aurora-gray-300 focus:outline-none focus:ring-1 focus:ring-aurora-blue-500"
-        />
-        <svg
-          className="absolute left-3 top-2.5 h-4 w-4 text-aurora-gray-500"
-          xmlns="http://www.w3.org/2000/svg"
-          fill="none"
-          viewBox="0 0 24 24"
-          stroke="currentColor"
+  <Stack
+    direction="vertical"
+    gap="4"
+    className={`filters   bg-theme-background-lvl-1
```

Comment:
format problems ? could you extract it and use maybe cn tool ?

Reply:
Here can be directly be used a string...